x-pack/legacy/plugins/index_management/common/constants/invalid_characters.ts

```diff
@@ -5,3 +5,5 @@
  */
 
 export const INVALID_INDEX_PATTERN_CHARS = ['\\', '/', '?', '"', '<', '>', '|'];
```

Comment:
see comment `template_form_schema.ts`. if you agree, i think we can remove this constant.

Reply:
See my EDIT above, I could not import it from 'ui/index_patterns'